### PR TITLE
Create CounterForMessenger.yml

### PR DESCRIPTION
**Pull request by Kubis10**

### CODE REVIEW

**Review by Silvrbckw — APPROVED** [review]

File: _data/projects/CounterForMessenger.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  - counter
+  - statistics
+  - facebook
+  - messenger
+upforgrabs:
+  name: Up-for-Grabs
+  link: https://github.com/Kubis10/CounterForMessenger/labels/Up-for-Grabs
+stats:
+  issue-count: 5
+  last-updated: '2022-07-30T22:25:24Z'
```
Comment (shiftkey) [inline _data/projects/CounterForMessenger.yml:16]: This will be generated through our automation at a later stage

```suggestion
```

**Review by shiftkey — APPROVED** [review]

File: _data/projects/CounterForMessenger.yml (hunk 1 above)
Comment (shiftkey) [inline _data/projects/CounterForMessenger.yml:16]:
```suggestion
  link: https://github.com/Kubis10/CounterForMessenger/labels/Up-for-Grabs
```

**Comment by shiftkey** [comment]
@Silvrbckw I'm not able to merge into your fork to get the missing CI checks passing, but otherwise this is good to go

**Comment by shiftkey** [comment]
Closing this out due to inactivity - if you're able to merge in the latest changes please re-submit and we can get this landed

**Pull request closed without merging**